app/controllers/application_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  # def current_user
+  #   User.first
+  # end
```

Comment:
- [ ] Please kindly remove this commented lines of code in order to make your codebase clean, ✔️